dallinger/experiment_server/experiment_server.py

```diff
@@ -536,16 +536,28 @@ def assign_properties(thing):
     session.commit()
 
 
-@app.route("/participant/<worker_id>/<hit_id>/<assignment_id>/<mode>",
+@app.route("/participant/<fingerprint_hash>/<worker_id>/<hit_id>/<assignment_id>/<mode>",
```

Comment:
Can you move `fingerprint_hash` into a query-string parameter or POST data to maintain backwards compatibility with existing experiments? The `create_participant` function will then look at the request arguments and, if no fingerprint hash is found, go ahead and create the participant.